rviz_visual_testing_framework/src/visual_test_fixture.cpp

```diff
@@ -29,6 +29,8 @@
 
 #include "rviz_visual_testing_framework/visual_test_fixture.hpp"
 
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <ament_index_cpp/get_package_prefix.hpp>
```

Comment:
This is still not quite correct, I'm afraid. They were in the correct block (below system includes), but are still not in alphabetical order... `prefix` should come before `share`...

Reply:
For order of includes see: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Noted. Updated accordingly. I wish I could use ```ament_clang_format --reformat visual_test_fixture.cpp``` which would take care of all this, but the rest of the file would be reformatted affecting history

Reply:
You should not use `ament_clang_format` as it is not up-to-date:

https://github.com/ament/ament_lint/blob/e08e905350821705353fcfc6bd561eb1a52ff895/ament_lint_common/package.xml#L15

`ament_uncrustify` should be used instead, but it doesn't cover include order. `ament_cpplint` will tell you if it's right but cannot reformat them for you.

Reply:
Thanks. I must have missed it.